docs/sources/features/datasources/testdata.md

```diff
@@ -50,5 +50,5 @@ Otherwise the dashboard will not be updated automatically for other Grafana user
 
 ## Using test data in issues
 
-If you post an issue on github regarding time series data or rendering of time series data we strongly advice you to use this data source to replicate the data.
+If you post an issue on github regarding time series data or rendering of time series data, we strongly advise you to use this data source to replicate the data.
```

Comment:
GitHub, not github

Reply:
@NiteshKoushik, can you make the suggested change. I can then merge the PR. Thanks!

Reply:
Hello @achatterjee-grafana , changes made as per suggestion.